lua/wire/wireshared.lua

```diff
@@ -999,3 +999,71 @@ function nicenumber.nicetime( n )
 		return "0s"
 	end
 end
+
+------------------------------------------------------
+-- Loads a font into the cache if it doesn't exist
+-- Returns true if font exists, else returns false
+------------------------------------------------------
+local _fontcache = {}
+local _defaultFontSize = 18
+
+function WireLib.LoadFont( name, size )
+	if not name then return nil end
+	
+	local fontSize = size or _defaultFontSize
+	local fontKey = name .. "_" .. fontSize
```

Comment:
Given that the logic for creating font keys could easily change (eg. it's already been suggested there should be a table parameter, so it would have to deal with parameters other than just name and size), the font key logic should be separated out into a local function rather than being repeated throughout every single function.